components/grid/columns/menu.md

```diff
@@ -12,34 +12,31 @@ position: 20
 
 The Grid allows you to setup a menu for its columns. It enables you to perform high-level customization like [sorting]({%slug components/grid/features/sorting%}), [filtering]({%slug components/grid/filtering%}), [showing or hiding]({%slug grid-columns-visible%}) columns and [freezing or unfreezing]({%slug grid-columns-frozen%}) them.
 
->caption Telerik Column Menu for the Grid
-
-![column menu basic example screenshot](images/column-menu-basic-example.png)
-
-In this article:
+>caption In this article:
 * [Basics](#basics)
 * [Features](#features)
     * [Sorting](#sorting)
     * [Filtering](#filtering)
     * [Frozen Columns](#frozen-columns)
     * [Column Chooser](#column-chooser)
     * [Sections](#sections)
+*[Example](#example)
 * [Notes](#notes)
 
 ## Basics
 
-To enable the column menu set the `ShowColumnMenu` parameter of the `<TelerikGrid>` tag to `true`. This will enable the menu for each column of the Grid.
+To enable the column menu, set the `ShowColumnMenu` parameter of the `<TelerikGrid>` tag to `true`. This will enable the menu for each column of the Grid.
```

Comment:
```suggestion
To enable the Column Menu, set the `ShowColumnMenu` parameter of the `<TelerikGrid>` tag to `true`. This will enable the menu for each column of the Grid.
```